apmpackage/apm/data_stream/app_metrics/fields/fields.yml

```diff
@@ -34,6 +34,26 @@
   type: long
   description: Current data collection period for this event in milliseconds.
   unit: ms
+- name: network.carrier.icc
+  type: keyword
+  description: |
+    ISO country code, eg. US
+- name: network.carrier.mcc
+  type: keyword
+  description: |
+    Mobile country code
+- name: network.carrier.mnc
+  type: keyword
+  description: |
+    Mobile network code
+- name: network.carrier.name
+  type: keyword
+  description: |
+    Carrier name, eg. Vodafone, T-Mobile, etc.
+- name: network.connection_type
+  type: keyword
+  description: |
+    Cellular network technology, eg. 4G
```

Comment:
We have already a top level `host` field, I didn't think it would make much sense to duplicate the field inside `network` to refer to the same host? 
`network` is already defined as top level field in ECS (https://www.elastic.co/guide/en/ecs/current/ecs-network.html),

I sort of followed https://github.com/elastic/apm-server/pull/491, btw.

Reply:
ECS says (about the network fieldset):

> The network is defined as the communication path over which a host or network event happens.
>
> The network.* fields should be populated with details about the network activity associated with an event.

Seems to me it makes sense to go under `network`

Reply:
After some offline discussion, `network.carrier.*` is where the new fields fit best related to ECS, and `net.host.carrier.*` was the prefered nesting in the open-telemetry proposal; so 👍 on the changes.